Coalition Folklore: Saryds (#5840)
endless-sky/endless-sky: data/coalition/coalition missions.txt

				accept
			label noship
			`	"Ah, I see. Well, just a reference it would be, so the name matters not, I suppose. Anyway, thanks again Captain."`

mission "Coalition Folklore: Saryd"
	minor
	invisible
	to offer
		random < 3
	source
		attributes "saryd" "folklore"
	on offer
		conversation

> 			`Walking by the spaceport, you're approached by a Saryd in unusually traditional clothing, even by Saryd standards. Silk robes not too different from human kimonos cover most of his body, and thin ropes tightly wrap around the fabric near his hoofs.`

Author: The mission is offered in the spaceport, if that's what you're asking. You meet the Saryd there and he brings you out to a forest if you agree to go with him to the festival. Should I change the "by" in the first sentence to "around" or "through" or something else?

> 			`Walking by the spaceport, you're approached by a Saryd in unusually traditional clothing, even by Saryd standards. Silk robes not too different from human kimonos cover most of his body, and thin ropes tightly wrap around the fabric near his hoofs.`

Reviewer: ```suggestion
			`Walking through the spaceport, you're approached by a Saryd in unusually traditional clothing, even by Saryd standards. Silk robes not too different from human kimonos cover most of his body, and thin ropes tightly wrap around the fabric near his hoofs.`
```
There you go. Hopefully that works the same way as in the review.

> 			`Walking by the spaceport, you're approached by a Saryd in unusually traditional clothing, even by Saryd standards, with silk robes not too different from human kimonos covering most of his body, and thin ropes tightly wrapping around it near his hoofs.`

Reviewer 2: This feels like it should be broken into two sentences.
I also find the end confusing. What does "it" refer to?

Author: It refers to the clothing, though now that you pointed that out, "wrapping around them" or "wrapping around the robes" would probably be better.
Where are you suggesting the break into two sentences?

			`	"Greetings, Captain. Interested in participating in a cultural celebration, would you be?" He asks.`
			choice
				`	"What sort of celebration?"`
				`	"Sorry, I'm busy right now."`
					decline
			`	"A celebration of Saryd folklore, it is," he explains. "Though for a long time among the stars we've lived, timeless classics from before our spacefaring age, we still celebrate. A set of theatrical plays about some of them, today we are hosting. Watch them for free, anyone can."`
			choice
				`	"I've always got time for a story, lead the way."`
				`	"I'm not much of a fan of plays, I'll pass."`
					decline
			`	You follow him outside the spaceport city, and deep into a nearby forest. On the way there, several other Saryds in ancient garbs are guiding others who were also intrigued by the offer.`

> 			`	You arrive at a clearing with three different paths to take, and you can make out a faint light deep within each of them. After a few minutes it seems as if all the Saryds involved in the celebration have arrived, as they grab three caskets from behind a bush, and lay them before you and the others they brought here.`

Reviewer 3: ```suggestion
			`	You arrive at a clearing with three different paths to take, and you can make out a faint light deep within each of them. After a few minutes it seems as if all the Saryds involved in the celebration have arrived. They grab three caskets from behind a bush, and lay them before you and the others they brought here.`
```

			`	"Now friends, choose your path, you must," one of the Saryds says, and many from the crowd grab small objects from the caskets, seemingly already familiar with this. You move forward to look into the caskets, and see that each one has different things inside: the left one is full of seeds, the middle one full of seashells, and the right one full of feathers.`
			choice
				`	(Grab a seed.)`
					goto seed
				`	(Grab a seashell.)`
					goto pearl
				`	(Grab a feather.)`

> 			`	Once you have the feather in your hands, the organizers tell you to go through the path on the right, and so you do.`

Author: In this the player character is going with the flow, so as it is clear that what you choose from the baskets determine what show you end up watching, they follow the instructions provided by the Saryd organizers.

> 			`	Once you have the feather in your hands, the organizers tell you to go through the path on the right, and so you do.`

Reviewer: Yes, I got that.
Basically, what I mean is this.

"the organizers tell you to go through the path, and so you do" is "You do as you are told."
"the organizers tell you to go through the path, and you do so" is "You do this thing, just like was said."

It's definitely not a huge thing, though. It's just a question of how you want it to feel. Whether you're "doing the same as everybody else" or "doing what you were told".

> 			`	The light you had seen turns out to be a bonfire, and you find a spot to sit on one of the logs close to it, while keeping your eyes on the small stage in front of you. After the audience is all here, the curtains open, to reveal many small figures made out of what looks like paper, in a style that loosely resembles origami.`

Reviewer 4: So I got through two of these and then it became clear that questions need to be asked.
Is this like a puppet show? How big exactly is this stage? A bonfire in the middle of a clearing is going to rather limit the field of space where the audience can get a view from, so is this a shadow play? 
There's a lot more that can be done here to visually set up the scene for the reader.

> 			`	Entering a clearing, a Saryd huntress with gray hair stalks a large bird resting on a rock. She moves on the borders of the clearing, hiding behind bushes and small trees, looking for the best spot from which to snipe at the bird, her bow and arrow ready in her hands. It's only when she is nearing a large entrance to the clearing, on the opposite side of where she entered, that she finds the spot, and takes aim.`

Reviewer 3: ```suggestion
			`	Entering a clearing, a Saryd huntress with gray hair stalks a large bird resting on a rock. She moves on the borders of the clearing, hiding behind bushes and small trees. She looks for the best spot from which to snipe at the bird, her bow and arrow ready in her hands. It's only when she is nearing a large entrance to the clearing, on the opposite side of where she entered, that she finds the spot, and takes aim.`
```

> 			`	Entering a clearing, a Saryd huntress with gray hair stalks a large bird resting on a rock. She moves on the borders of the clearing, hiding behind bushes and small trees. She looks for the best spot from which to snipe at the bird, her bow and arrow ready in her hands. It's only when she is nearing a large entrance to the clearing, on the opposite side of where she entered, that she finds the spot, and takes aim.`

Reviewer 3: I think this paragraph has the same issue the whale in the seashell story had. When I read that a Saryd was hunting a bird, I picture a hawk or some other obviously Earth animal. Maybe change it so that it's clear that the Saryd is hunting some bird-like alien animal, and not a literal bird from Earth.

Reviewer: I'd actually have to disagree with this. A "bird" is a generic classification, where a "whale" is a specific creature.

Reviewer 3: Hmm, but 'bird' is a Earth classification of animal. Alien animals wouldn't fit into Earth biology. The most that they could do is somewhat resemble birds. 
Maybe I'm just being really nitpicky. :P

Reviewer: We could just start calling everything "bird-like", "beetle-like", or "spider like" but it would get cumbersome, and probably annoying to read, pretty quickly. While I can understand arguments for not calling alien creatures by specific human animal names, I think when it general classification suchs as "bird", "fish", "insect" it's perfectly exceptable to just go with it. The average player isn't going to want a biology lesson in the story, those are saved for logbook entries.

Reviewer 5: May I suggest "avian" or "avian creature". Not as short as "bird", but perhaps adds some variation and leaves a bit more room for interpretation.

Author: I don't think it's worth describing the bird, in Coalition planet descriptions there are cases of "shoals of fish" or "spiders so large that their webs are strung from one tree to another," so it's not like every single instance of an animal being mentioned needs to be accompanied by a description to show how it's not an exact copy of an Earth animal, so I'll just leave this one as it is.

Reviewer 3: Fair.

> 			`	She is disrupted when she hears a loud roar, and jumps to the ground to narrowly avoiding the beast that had come from behind her. You're unsure how accurate the paper's depiction of the creature is, but you can only describe it as a hedgelion: a bit larger than lions from Earth, with various quills where its mane should be. The quills continue in a line on its back, reaching the tip of the tail.`

Reviewer: ```suggestion
			`	She is disrupted when she hears a loud roar, and jumps to the ground, narrowly avoiding the beast that had come from behind her. You're unsure how accurate the paper's depiction of the creature is, but you can only describe it as a hedgelion: a bit larger than lions from Earth, with various quills where its mane should be. The quills continue in a line on its back, reaching the tip of the tail.`
```

Oh hell. How did I make this mistake? I'm embarassed. Good thing I looked over this again.

> 			`	She is disrupted when she hears a loud roar, and jumps to the ground to narrowly avoiding the beast that had come from behind her: you're unsure how accurate the paper's depiction of the creature is, but you can only describe it as a hedgelion, a bit larger than lions from Earth, and with various quills where its mane should be. The quills continue in a line on its back, reaching the tip of the tail.`

Reviewer 3: ```suggestion
			`	She is disrupted when she hears a loud roar, and jumps to the ground to narrowly avoiding the beast that had come from behind her. You're unsure how accurate the paper's depiction of the creature is, but you can only describe it as a hedgelion: a bit larger than lions from Earth, with various quills where its mane should be. The quills continue in a line on its back, reaching the tip of the tail.`
```

			`	Having dodged the pounce, she runs past the bird, looking for cover, but the forest isn't dense enough to block the beast's advance. It attacks again, and wounds the top of one of her frontal legs. She kicks it with another leg, and readies an arrow, but it only scratches its face. The animal gets on its hind legs to swipe down at her, but is stunned, for the bird has attacked its eyes, blinding it. The huntress is swift to take aim, and an arrow pierces the creature's torso. Once it's fallen the bird flies away, and the curtains close.`
				goto end
			label seed
			`	Once you have the seed in your hands, the organizers tell you to go through the path on the left, and so you do.`
			`	The light you had seen turns out to be a bonfire, and you find a spot to sit on one of the logs close to it, while keeping your eyes on the small stage in front of you. After the audience is all here, the curtains open, to reveal many small figures made out of what looks like paper, in a style that loosely resembles origami.`

> 			`	The central figure is a young Saryd boy, who is galloping in a dense forest, as the sound effects of thunder grow increasingly frequent. Lightning strikes one of the trees on the way, and it falls in flames, blocking the boy's path, so he turns to run elsewhere. Another lightning bolt, and another tree falls. After a few more repetitions, the boy is too tired to run anymore, hurt by some of the flames and now limping through the forest.`

Reviewer 4: Is this actual lightning? Or theatrics? Following on from the previous one. If these things are mostly made from some paper equivalent and its not actual Saryd in costumes, how is the effect of burning and falling trees simulated? 
Even more than the previous one, this needs the scene to be set in greater detail because lightning and trees falling in the play, when you're sitting in an actual forest clearing, makes it feel very confusing and unclear what is happening and how it is doing so.

> 			`	Whimpering as the thunder grows closer once more, he spots a tree unlike the others, a great oak with branches thicker than the trees around him, so he goes to it. Laying down at the base of that largest tree, he falls asleep, and the curtain closes.`

Reviewer 4: If this is not live-action with actual Saryd, and is using paper figures, then you wouldn't need to close the curtains to set the branches in place, it would make more sense for the branches protecting the boy to happen stylistically. This would be a very odd curtain use in the middle of what is already a very short play.

Author: The curtains closing are meant to signify that the boy has lost consciousness, and serve as a transition for the time he's asleep.

Reviewer 4: Yes I know, but that's only something that's usually done with live actors, when people might have to enter the stage to rearrange things etc. For other styles of play simple blackouts or other stylistic methods of demonstrating the passage of time are more typical.
Of course, you are welcome to keep it as a curtain close. It just seems especially odd since it has literally only two scenes.

> 			`	The central figure is a young Saryd boy, galloping in a dense forest. Sound effects of thunder begin and grow increasingly frequent, echoing around you as if in a real thunderstorm. A white piece of paper, shaped like a lightning bolt, comes down swiftly and strikes one of the trees in the boy's way. Flame-shaped folds rain down from the top of the stage and stick to the tree as it falls, blocking the boy's path. He turns to run elsewhere. Another lightning bolt, and another tree falls. After a few more repetitions, the boy is too tired to run anymore, hurt by some of the flames and now limping through the forest.`

Reviewer: ```suggestion
			`	The central figure is a young Saryd boy, galloping through a dense forest. Sound effects of thunder begin and grow increasingly frequent, echoing around you as if in a real thunderstorm. A white piece of paper, shaped like a lightning bolt, comes down swiftly and strikes one of the trees in the boy's way. Flame-shaped folds rain down from the top of the stage and stick to the tree as it falls, blocking the boy's path. He turns to run elsewhere. Another lightning bolt, and another tree falls. After a few more repetitions, the boy is too tired to run anymore. Hurt by some of the flames, he is now limping through the forest.`
```
Improved readability of introduction, and added dramatic effect to the end.

> 			`	The central figure is a young Saryd boy, who is galloping in a dense forest, as the sound effects of thunder grow increasingly frequent, echoing around you as if in a real thunderstorm. A white piece of paper, shaped like a lightning bolt, comes down swiftly and strikes one of the trees in the boy's way. Flame-shaped folds rain down from the top of the stage and stick to the tree as it falls, blocking the boy's path. He turns to run elsewhere. Another lightning bolt, and another tree falls. After a few more repetitions, the boy is too tired to run anymore, hurt by some of the flames and now limping through the forest.`

Reviewer 3: ```suggestion
			`	The central figure is a young Saryd boy, galloping in a dense forest. Sound effects of thunder begin and grow increasingly frequent, echoing around you as if in a real thunderstorm. A white piece of paper, shaped like a lightning bolt, comes down swiftly and strikes one of the trees in the boy's way. Flame-shaped folds rain down from the top of the stage and stick to the tree as it falls, blocking the boy's path. He turns to run elsewhere. Another lightning bolt, and another tree falls. After a few more repetitions, the boy is too tired to run anymore, hurt by some of the flames and now limping through the forest.`
```

> 			`	Whimpering as the thunder grows closer once more, he spots a tree unlike the others, a great oak with branches thicker than the trees around him. He goes to it and lays down at the base of that largest tree, and soon he falls asleep as the curtain closes.`

Reviewer: ```suggestion
			`	Whimpering as the thunder grows closer once more, he spots a tree unlike the others, a great oak with branches thicker than the trees around him. He goes to it and lays down at its base, and soon he falls asleep as the curtain closes.`
```
An alternative  would be "at the base of that largest of trees". The suggestion improves readability, the comment emphasizes that the tree is special.

> 			`	Whimpering as the thunder grows closer once more, he spots a tree unlike the others, a great oak with branches thicker than the trees around him, so he goes to it. Laying down at the base of that largest tree, he falls asleep, and the curtain closes.`

Reviewer 3: ```suggestion
			`	Whimpering as the thunder grows closer once more, he spots a tree unlike the others, a great oak with branches thicker than the trees around him. He goes to it and lays down at the base of that largest tree, and soon he falls asleep as the curtain closes.`
```

			`	It opens once again after a few moments, with the boy waking up in darkness. When he gets up, rumbling sounds are accompanie by rays of light reaching the ground, and he realizes the great oak had lowered its branches around him, and as it lifts them up once again, it knocks back several trees that had fallen on them during the storm. The sun as his guide, the boy manages to leave the forest, and the curtains close again.`
				goto end
			label pearl
			`	Once you have the seashell in your hands, the organizers tell you to go through the path on the middle, and so you do.`
			`	The light you had seen turns out to be a bonfire, and you find a spot to sit on one of the logs close to it, while keeping your eyes on the small stage in front of you. After the audience is all here, the curtains open, to reveal many small figures made out of what looks like paper, in a style that loosely resembles origami.`

> 			`	A Saryd is on a boat, fishing at sea. He feels a bite, but once he reels it in there is nothing on the hook. He feels another bite, and again finds nothing. Time and again, he looks to the hook to find the bait still there, intact, but continues to feel the bites, that are sometimes so blatant even the boat moves a little. The day moves by quickly, with a piece of paper representing the sun moving swiftly from one side of the "sky" to the other.`

Reviewer 4: This is another place where <First> is unlikely to have the needed experience to grasp this instantly. For simple biomechanical reasons a Saryd would be unable to use a small boat in the human style. So if they have small individual boats they would look quite different indeed to our conception of a small fishing boat.
Personally I don't think Saryds would use individual boats, but if they're going to then some thought needs to be put into describing the boat, because a small Saryd boat probably wouldn't be recognisable to a human as being a boat without a lot of context that would be hard to express through origami.

> 			`	Once you have the seashell in your hands, the organizers tell you to go through the path on the middle, and you do so.`

Reviewer: ```suggestion
			`	Once you have the seashell in your hands, the organizers tell you to go through the path in the middle, and you do so.`
```

> 			`	A Saryd is on top of some sort of long, wooden platform, with a deep blue background under it. Only when you see the fishing rod do you realize that he is on a boat, fishing at sea. He feels a bite, but once he reels it in there is nothing on the hook. He feels another bite, and again finds nothing. Time and again, he looks to the hook to find the bait still there, intact, yet he continues to feel bites that are sometimes so blatant even the boat moves a little. The day moves by quickly, with a piece of paper representing the sun moving swiftly from one side of the "sky" to the other.`

Reviewer: ```suggestion
			`	A Saryd is on top of some sort of long, wooden platform, with a deep blue background under it. Only when you see the fishing rod do you realize that he is on a boat, fishing at sea. He feels a bite, but when he reels it in there is nothing on the hook. He feels another bite, and again finds nothing. Time and again, he looks at the hook to find the bait still there, intact, yet he continues to feel bites that are sometimes so blatant even the boat moves a little. The day moves by quickly, with a piece of paper representing the sun moving swiftly from one side of the "sky" to the other.`
```

> 			`	Once it is night, the waves grow increasingly stronger, and the Saryd stops fishing altogether, putting the rod on the boat and trying to steer it back to shore, but he can't find the shore. A huge wave that goes beyond the stage's top approaches, and engulfs the boat.`

Reviewer 3: Some sentences are quite long and unwieldy. Also, `steer it back to shore, but he can't find the shore.` sounds kind of repetitive. 
```suggestion
			`	Once it is night, the waves grow increasingly stronger, and the Saryd stops fishing altogether. He puts the rod on the boat and tries to steer it back to shore, but he can't find his way. A huge wave that goes beyond the stage's top approaches, and engulfs the boat.`
```

> 			`	The fisherman kicks and strugles for air, having lost his boat and still at the mercy of the great waves. The view on the stage starts going down, following the kicks, and something starts rising up to meet the Saryd's legs. As it raises the view back with it, a whale comes to the surface, gently and with care, as if trying to serve as ground for the sailor. Another wave is approaching, large enough to engulf them both, but the whale keeps on going up, until it is out of the water.`

Reviewer 3: This is very good writing, and the imagery is great. However, it feels a bit weird to me that a whale, an Earth animal, would be saving a Saryd in one of their ancient stories. Maybe change it to make it clear that it's not a literal whale that we're seeing, but rather an alien animal that happens to look somewhat like an Earth whale.

> 			`	As it flies through the night sky, the whale is careful to not drop the Saryd, always waiting for him to find stable footing before turning. For the rest of the night, the whale slowly dances in the sky, and when morning comes and the waves subside, it goes back to the sea, and swims close to shore. Once the Saryd is at the beach, he turns back to see the whale's tail hitting the water as it dives deep once again, and the curtains close.`

Reviewer 3: ```suggestion
			`	As it flies through the night sky, the whale is careful to not drop the Saryd, always waiting for him to find stable footing before turning. For the rest of the night, the whale slowly dances in the sky, and when morning comes and the waves subside, it goes back to the sea and swims close to shore. Once the Saryd is at the beach, he turns back to see the whale's tail hitting the water as it dives deep once again, and the curtains close.`
```

				accept
			label noship
			`	"Ah, I see. Well, just a reference it would be, so the name matters not, I suppose. Anyway, thanks again Captain."`

mission "Coalition Folklore: Saryd"
	minor
	invisible
	to offer
		random < 30
	source

> 		attributes "saryd" "folklore"

Reviewer 6: Same situation as the news locations. I'm assuming you want this to offer on a planet with "saryd" *and* "folklore," in which case you need two attribute lines. I imagine you'll want to look at the other folklore missions for this same error.
```suggestion
		attributes "saryd"
		attributes "folklore"
```

	on offer
		conversation
			`Walking through the spaceport, you're approached by a Saryd in unusually traditional clothing, even by Saryd standards. Silk robes not too different from human kimonos cover most of his body, and thin ropes tightly wrap around the fabric near his hoofs.`

> 			`	"Greetings, Captain. Interested in participating in a cultural celebration, would you be?" He asks.`

Reviewer 7: ```suggestion
			`	"Greetings, Captain. Interested in participating in a cultural celebration, would you be?" he asks.`
```
Here's a quick rule with dialogue, if you're interested: if you're finishing with a question/exclamation mark (like you've done here), only capitalize the next word if it starts a new sentence. In this case, `he says` is finishing the sentence which includes the dialogue before it. It's a little bit of a strange case.
I have this strange feeling that I saw this somewhere else recently and didn't correct it. It's probably not worth digging for, but if anything comes to mind, let me know.

			choice
				`	"What sort of celebration?"`
				`	"Sorry, I'm busy right now."`
					decline
			`	"A celebration of Saryd folklore, it is," he explains. "Though for a long time among the stars we've lived, timeless classics from before our spacefaring age, we still celebrate. A set of theatrical plays about some of them, today we are hosting. Watch them for free, anyone can."`
			choice

> 				`	"I've always got time for a story, lead the way."`
> 				`	"I'm not really up for a story now, I'll pass."`

Reviewer 7: ```suggestion
				`	"I've always got time for a story. Lead the way."`
				`	"I'm not really up for a story now. I'll pass."`
```
You could also make them semicolons or do a mix of the two.

					decline

> 			`	You follow him outside the spaceport city, and deep into a nearby forest. On the way there, several other Saryds in ancient garbs are guiding others who were also intrigued by the offer.`

Reviewer 7: ```suggestion
			`	You follow him outside the spaceport city and deep into a nearby forest. On the way there, several other Saryds in ancient garbs are guiding others who were also intrigued by the offer.`
```

> 			`	You arrive at a clearing with three different paths to take, and you can make out a faint light deep within each of them. After a few minutes it seems as if all the Saryds involved in the celebration have arrived. They grab three caskets from behind a bush, and lay them before you and the others they brought here.`

Reviewer 7: ```suggestion
			`	You arrive at a clearing with three different paths to take, and you can make out a faint light deep within each of them. After a few minutes, it seems as if all the Saryds involved in the celebration have arrived. They grab three caskets from behind a bush and lay them before you and the others they brought here.`
```

> 			`	"Now friends, choose your path, you must," one of the Saryds says, and many from the crowd grab small objects from the caskets, seemingly already familiar with this. You move forward to look into the caskets, and see that each one has different things inside: the left one is full of seeds, the middle one full of seashells, and the right one full of feathers.`

Reviewer 7: ```suggestion
			`	"Now friends, choose your path, you must," one of the Saryds says, and many from the crowd grab small objects from the caskets, seemingly already familiar with this. You move forward to look into the caskets and see that each one has different things inside: the left one is full of seeds, the middle one full of seashells, and the right one full of feathers.`
```

			choice
				`	(Grab a seed.)`
					goto seed
				`	(Grab a seashell.)`
					goto pearl
				`	(Grab a feather.)`

> 			`	Once you have the feather in your hands, the organizers tell you to go through the path on the right, and you do so.`

Reviewer 6: "and you do so" feels unnecessary to me. I think leaving it off sounds better. I think it can be safely assumed that you take the path you were pointed to without extra detail saying that you do so.
```suggestion
			`	Once you have the feather in your hands, the organizers tell you to go through the path on the right.`
```
This same change would be made to the other two choices.

> 			`	The light you had seen turns out to be a bonfire, and you find a spot to sit on one of the logs close to it. You keep your eyes on the wooden stage in front of you, about a third of the size of human opera stages. Several lights shine onto it, and as you look up you can only barely see some spotlights high up in the trees. After the audience is all here, the curtains open to reveal many small figures made out of what looks like paper in a style that loosely resembles origami. They are supported either by thin wooden poles from below, or strings from above. There are figures close to where the curtains were, and others further into the stage, the various layers granting depth to the scenery. After a brief moment, the show begins.`

Reviewer 7: ```suggestion
			`	The light you had seen turns out to be a bonfire, and you find a spot to sit on one of the logs close to it. You keep your eyes on the wooden stage in front of you, about a third of the size of human opera stages. Several lights shine onto it, and as you look up, you can only barely see some spotlights high up in the trees. After the audience is all here, the curtains open to reveal many small figures made out of what looks like paper in a style that loosely resembles origami. They are supported either by thin wooden poles from below or strings from above. There are figures close to where the curtains were and others further into the stage, the various layers granting depth to the scenery. After a brief moment, the show begins.`
```

			`	Entering a clearing, a Saryd huntress with gray hair stalks a large bird resting on a rock. She moves on the borders of the clearing, hiding behind bushes and small trees. She looks for the best spot from which to snipe at the bird, her bow and arrow ready in her hands. It's only when she is nearing a large entrance to the clearing, on the opposite side of where she entered, that she finds the spot, and takes aim.`

> 			`	She is disrupted when she hears a loud roar, and jumps to the ground, narrowly avoiding the beast that had come from behind her. You're unsure how accurate the paper's depiction of the creature is, but you can only describe it as a hedgelion: a bit larger than lions from Earth, with various quills where its mane should be. The quills continue in a line on its back, reaching the tip of the tail.`

Reviewer 7: ```suggestion
			`	She is disrupted when she hears a loud roar and jumps to the ground, narrowly avoiding the beast that had come from behind her. You're unsure how accurate the paper's depiction of the creature is, but you can only describe it as a hedge lion: a bit larger than lions from Earth, with various quills where its mane should be. The quills continue in a line on its back, reaching the tip of the tail.`
```
Is a `hedgelion` just a lion made from trimming a hedge/bush? If so, I think it needs a space. If not, I'm not too sure what we're talking about.

> 			`	Having dodged the pounce, she runs past the bird, looking for cover, but the forest isn't dense enough to block the beast's advance. It attacks again, and wounds the top of one of her frontal legs. She kicks it with another leg, and readies an arrow, but it only scratches its face. The animal gets on its hind legs to swipe down at her, but is stunned, for the bird has attacked its eyes, blinding it. The huntress is swift to take aim, and an arrow pierces the creature's torso. Once it's fallen the bird flies away, and the curtains close.`

Reviewer 7: ```suggestion
			`	Having dodged the pounce, she runs past the bird, looking for cover, but the forest isn't dense enough to block the beast's advance. It attacks again, and wounds the top of one of her frontal legs. She kicks it with another leg and readies an arrow, but it only scratches its face. The animal gets on its hind legs to swipe down at her but is stunned, for the bird has attacked its eyes, blinding it. The huntress is swift to take aim, and an arrow pierces the creature's torso. Once it's fallen, the bird flies away, and the curtains close.`
```

				goto end
			label seed
			`	Once you have the seed in your hands, the organizers tell you to go through the path on the left, and you do so.`

> 			`	The light you had seen turns out to be a bonfire, and you find a spot to sit on one of the logs close to it. You keep your eyes on the wooden stage in front of you, about a third of the size of human opera stages. Several lights shine onto it, and as you look up, you can only barely see some spotlights high up in the trees. After the audience is all here, the curtains open to reveal many small figures made out of what looks like paper in a style that loosely resembles origami. They are supported either by thin wooden poles from below or strings from above. There are figures close to where the curtains were and others further into the stage, the various layers granting depth to the scenery. After a brief moment, the show begins.`

Reviewer 8: ```suggestion
			`	The light you had seen turns out to be a bonfire near a wooden stage, about a third of the size of human opera stages, and you find a spot to sit on one of the logs positioned near the fire. Several lights shine onto the stage, and as you look up, you can only barely see some spotlights hidden high up in the trees. After the audience is all gathered, the curtains of the stage open to reveal many small figures made out of what looks like paper in a style that loosely resembles origami. They are supported either by thin wooden poles from below or strings from above. There are figures close to where the curtains were and others further away on the stage, the various layers granting depth to the scenery. After a brief moment, the show begins:`
```

> 			`	The light you had seen turns out to be a bonfire, and you find a spot to sit on one of the logs close to it. You keep your eyes on the wooden stage in front of you, about a third of the size of human opera stages. Several lights shine onto it, and as you look up you can only barely see some spotlights high up in the trees. After the audience is all here, the curtains open to reveal many small figures made out of what looks like paper in a style that loosely resembles origami. They are supported either by thin wooden poles from below, or strings from above. There are figures close to where the curtains were, and others further into the stage, the various layers granting depth to the scenery. After a brief moment, the show begins.`

Reviewer 7: ```suggestion
			`	The light you had seen turns out to be a bonfire, and you find a spot to sit on one of the logs close to it. You keep your eyes on the wooden stage in front of you, about a third of the size of human opera stages. Several lights shine onto it, and as you look up, you can only barely see some spotlights high up in the trees. After the audience is all here, the curtains open to reveal many small figures made out of what looks like paper in a style that loosely resembles origami. They are supported either by thin wooden poles from below or strings from above. There are figures close to where the curtains were and others further into the stage, the various layers granting depth to the scenery. After a brief moment, the show begins.`
```

			`	The central figure is a young Saryd boy, galloping through a dense forest. Sound effects of thunder begin and grow increasingly frequent, echoing around you as if in a real thunderstorm. A white piece of paper, shaped like a lightning bolt, comes down swiftly and strikes one of the trees in the boy's way. Flame-shaped folds rain down from the top of the stage and stick to the tree as it falls, blocking the boy's path. He turns to run elsewhere. Another lightning bolt, and another tree falls. After a few more repetitions, the boy is too tired to run anymore. Hurt by some of the flames, he is now limping through the forest.`

> 			`	Whimpering as the thunder grows closer once more, he spots a tree unlike the others, a great oak with branches thicker than the trees around him. He goes to it and lays down at its base, and soon he falls asleep as the curtain closes.`

Reviewer 8: ```suggestion
			`	Whimpering as the thunder grows closer once more, he spots a tree unlike the others, a great oak with branches thicker than the other trees around him. He goes to it and lays down at its base, and soon he falls asleep as the curtain closes.`
```

> 			`	It opens once again after a few moments, with the boy waking up in darkness. When he gets up, rumbling sounds are accompanied by rays of light reaching the ground, and he realizes the great oak had lowered its branches around him. As it lifts them up, it knocks back several trees that had fallen on them during the storm. The sun as his guide, the boy manages to leave the forest, and the curtains close again.`

Reviewer 8: ```suggestion
			`	It opens once again after a few moments, with the boy waking up in darkness. When he gets up, rumbling sounds are accompanied by rays of light reaching the ground, and he realizes the great oak had lowered its branches around him. As it lifts them up, the several smaller trees that had fallen on the mighty oak's branches during the storm are scattered away. The sun as his guide, the boy manages to leave the forest, and the curtains close again.`
```

				goto end
			label pearl
			`	Once you have the seashell in your hands, the organizers tell you to go through the path in the middle, and you do so.`

> 			`	The light you had seen turns out to be a bonfire, and you find a spot to sit on one of the logs close to it. You keep your eyes on the wooden stage in front of you, about a third of the size of human opera stages. Several lights shine onto it, and as you look up, you can only barely see some spotlights high up in the trees. After the audience is all here, the curtains open to reveal many small figures made out of what looks like paper in a style that loosely resembles origami. They are supported either by thin wooden poles from below or strings from above. There are figures close to where the curtains were and others further into the stage, the various layers granting depth to the scenery. After a brief moment, the show begins.`

Reviewer 8: ```suggestion
			`	The light you had seen turns out to be a bonfire near a wooden stage, about a third of the size of human opera stages, and you find a spot to sit on one of the logs positioned near the fire. Several lights shine onto the stage, and as you look up, you can only barely see some spotlights hidden high up in the trees. After the audience is all gathered, the curtains of the stage open to reveal many small figures made out of what looks like paper in a style that loosely resembles origami. They are supported either by thin wooden poles from below or strings from above. There are figures close to where the curtains were and others further away on the stage, the various layers granting depth to the scenery. After a brief moment, the show begins:`
```

> 			`	The light you had seen turns out to be a bonfire, and you find a spot to sit on one of the logs close to it. You keep your eyes on the wooden stage in front of you, about a third of the size of human opera stages. Several lights shine onto it, and as you look up you can only barely see some spotlights high up in the trees. After the audience is all here, the curtains open to reveal many small figures made out of what looks like paper in a style that loosely resembles origami. They are supported either by thin wooden poles from below, or strings from above. There are figures close to where the curtains were, and others further into the stage, the various layers granting depth to the scenery. After a brief moment, the show begins.`

Reviewer 7: ```suggestion
			`	The light you had seen turns out to be a bonfire, and you find a spot to sit on one of the logs close to it. You keep your eyes on the wooden stage in front of you, about a third of the size of human opera stages. Several lights shine onto it, and as you look up, you can only barely see some spotlights high up in the trees. After the audience is all here, the curtains open to reveal many small figures made out of what looks like paper in a style that loosely resembles origami. They are supported either by thin wooden poles from below or strings from above. There are figures close to where the curtains were and others further into the stage, the various layers granting depth to the scenery. After a brief moment, the show begins.`
```

> 			`	A Saryd is on top of some sort of long, wooden platform, with a deep blue background under it. Only when you see the fishing rod do you realize that he is on a boat, fishing at sea. He feels a bite, but when he reels it in, there is nothing on the hook. He feels another bite and again finds nothing. Time and again, he looks at the hook to find the bait still there, intact, yet he continues to feel bites that are sometimes so noticeable even the boat moves a little. The day moves by quickly, with a piece of paper representing the sun moving swiftly from one side of the "sky" to the other.`

Reviewer 8: ```suggestion
			`	A Saryd is on top of some sort of long, wooden platform, with a deep blue background under it. Only when you see the fishing rod do you realize that he is on a boat, fishing at sea. He feels a bite, but when he reels it in, there is nothing on the hook. He feels another bite and again finds nothing. Time and again, he looks at the hook to find the bait still there, intact, yet he continues to feel bites, some so powerful even the boat moves a little. The day moves by quickly, with a piece of paper representing the sun moving swiftly from one side of the "sky" to the other.`
```

> 			`	A Saryd is on top of some sort of long, wooden platform, with a deep blue background under it. Only when you see the fishing rod do you realize that he is on a boat, fishing at sea. He feels a bite, but when he reels it in there is nothing on the hook. He feels another bite, and again finds nothing. Time and again, he looks at the hook to find the bait still there, intact, yet he continues to feel bites that are sometimes so blatant even the boat moves a little. The day moves by quickly, with a piece of paper representing the sun moving swiftly from one side of the "sky" to the other.`

Reviewer 7: ```suggestion
			`	A Saryd is on top of some sort of long, wooden platform, with a deep blue background under it. Only when you see the fishing rod do you realize that he is on a boat, fishing at sea. He feels a bite, but when he reels it in, there is nothing on the hook. He feels another bite and again finds nothing. Time and again, he looks at the hook to find the bait still there, intact, yet he continues to feel bites that are sometimes so noticeable even the boat moves a little. The day moves by quickly, with a piece of paper representing the sun moving swiftly from one side of the "sky" to the other.`
```
I feel like `noticeable` or another word may be a little more appropriate for what you're trying to say rather than `blatant`. Just my take though

			`	Once it is night, the waves grow increasingly stronger, and the Saryd stops fishing altogether. He puts the rod on the boat and tries to steer it back to shore, but he can't find his way. A huge wave that goes beyond the stage's top approaches, and engulfs the boat.`
			`	The fisherman kicks and struggles for air, having lost his boat and still at the mercy of the great waves. The view on the stage starts going down, following the kicks, and something starts rising up to meet the Saryd's legs. As it raises the view back with it, a creature reminiscent of a whale, only with much larger, longer flippers that almost link to its tail, comes to the surface. It rises gently and with care, as if trying to serve as ground for the sailor. Another wave is approaching, large enough to engulf them both, but the whale keeps on going up until it is out of the water.`

> 			`	As it flies through the night sky, the whale is careful to not drop the Saryd, always waiting for him to find stable footing before turning. For the rest of the night, the whale slowly dances in the sky, and when morning comes and the waves subside, it goes back to the sea and swims close to shore. Once the Saryd is at the beach, he turns back to see the whale's tail hitting the water as it dives deep once again, and the curtains close.`

Reviewer 7: Wow. I really, _really_ love how the whale goes from unknown and intimidating to beautiful and majestic. I'm not sure why I like this so much, but something about this just resonates with me. This is a really cool moment. Well done.

			label end
			`	People in the audience start getting up and preparing to leave, and you follow them. Once you're back at the clearing, you return the small object back to its casket after seeing others do the same, and the organizers guide everyone back to the spaceport.`

> 			`	Once there, the Saryd who first approached you says "That you enjoyed the spectacle, I hope," and bids you farewell.`

Reviewer 7: ```suggestion
			`	Once there, the Saryd who first approached you says, "That you enjoyed the spectacle, I hope," and bids you farewell.`
```

> 			`	People in the audience start getting up and preparing to leave, and you follow them. Once you're back at the clearing, you return the small object back to its casket after seeing others do the same, and the organizers guide everyone back to the spaceport.`

Reviewer 6: Perhaps describe the audience applauding before getting up to leave. Something to show that the people enjoyed the play. Without that, I imagined the curtains closing and people simply standing up and leaving without any reaction; somewhat comical to me, but in any case not the feel that I think we're going for.

Author: Don't thing an actual round of applause would be appropriate for Saryds in this situation, so made it be a bowing gesture with hands together as if in a prayer. Does that work?

Reviewer 6: Looks good.

				goto end
			label seed
			`	Once you have the seed in your hands, the organizers tell you to go through the path on the left, and you do so.`
			`	The light you had seen turns out to be a bonfire, and you find a spot to sit on one of the logs close to it. You keep your eyes on the wooden stage in front of you, about a third of the size of human opera stages. Several lights shine onto it, and as you look up you can only barely see some spotlights high up in the trees. After the audience is all here, the curtains open to reveal many small figures made out of what looks like paper in a style that loosely resembles origami. They are supported either by thin wooden poles from below, or strings from above. There are figures close to where the curtains were, and others further into the stage, the various layers granting depth to the scenery. After a brief moment, the show begins.`

> 			`	The central figure is a young Saryd boy, who is galloping in a dense forest, as the sound effects of thunder grow increasingly frequent, echoing around you as if in a real thunderstorm. A white piece of paper comes down swiftly, as lightning strikes one of the trees in the boy's way, flame-shaped folds raining down from the top of the stage and sticking to it as it falls, blocking the boy's path, so he turns to run elsewhere. Another lightning bolt, and another tree falls. After a few more repetitions, the boy is too tired to run anymore, hurt by some of the flames and now limping through the forest.`

Reviewer 3: It's unclear to me what `sticking to it as it falls` means in this context. What is it sticking to? What is falling? Perhaps also make it clearer that the 'white piece of paper' is supposed to represent lightning, because the transition from `A white piece of paper comes down swiftly, as lightning strikes one of the trees on the boy's way...` seems a bit abrupt to me.

Author: The "flame-shaped folds (of paper)" are what falls, falling onto the tree struck by the lightning bolt (said tree is falling due to being struck by lightning), and the flames stick to it signifying it's on fire.
I needed something to introduce the fire to the tree so that it's not like the paper representing fire came out of nowhere, but the only other alternative I thought would be the paper tree spinning to its back, which would be modeled to already look on fire, but that would cause issues with the mental image as it spinning would hit the other layers of paper.

Reviewer 3: Fair enough. The changes you've made make what's going on much clearer.

			`	A Saryd is on top of some sort of long, wooden platform, with a deep blue background under it. Only when you see the fishing rod do you realize that he is on a boat, fishing at sea. He feels a bite, but once he reels it in there is nothing on the hook. He feels another bite, and again finds nothing. Time and again, he looks to the hook to find the bait still there, intact, yet he continues to feel bites that are sometimes so blatant even the boat moves a little. The day moves by quickly, with a piece of paper representing the sun moving swiftly from one side of the "sky" to the other.`
			`	Once it is night, the waves grow increasingly stronger, and the Saryd stops fishing altogether. He puts the rod on the boat and tries to steer it back to shore, but he can't find his way. A huge wave that goes beyond the stage's top approaches, and engulfs the boat.`

> 			`	The fisherman kicks and strugles for air, having lost his boat and still at the mercy of the great waves. The view on the stage starts going down, following the kicks, and something starts rising up to meet the Saryd's legs. As it raises the view back with it, a creature reminiscent of a whale, only with much larger, longer flippers that almost link to its tail, comes to the surface, gently and with care, as if trying to serve as ground for the sailor. Another wave is approaching, large enough to engulf them both, but the whale keeps on going up, until it is out of the water.`

Reviewer 3: Just noticed a typo in 'struggles'. Some sentences are also kind of long.
```suggestion
			`	The fisherman kicks and struggles for air, having lost his boat and still at the mercy of the great waves. The view on the stage starts going down, following the kicks, and something starts rising up to meet the Saryd's legs. As it raises the view back with it, a creature reminiscent of a whale, only with much larger, longer flippers that almost link to its tail, comes to the surface. It rises gently and with care, as if trying to serve as ground for the sailor. Another wave is approaching, large enough to engulf them both, but the whale keeps on going up until it is out of the water.`
```

> 			`	As it flies through the night sky, the whale is careful to not drop the Saryd, always waiting for him to find stable footing before turning. For the rest of the night, the whale slowly dances in the sky, and when morning comes and the waves subside, it goes back to the sea, and swims close to shore. Once the Saryd is at the beach, he turns back to see the whale's tail hitting the water, as it dives deep once again, and the curtains close.`

Reviewer 3: I feel like it would be more natural to remove a comma there, so sentence flow is better.
```suggestion
			`	As it flies through the night sky, the whale is careful to not drop the Saryd, always waiting for him to find stable footing before turning. For the rest of the night, the whale slowly dances in the sky, and when morning comes and the waves subside, it goes back to the sea, and swims close to shore. Once the Saryd is at the beach, he turns back to see the whale's tail hitting the water as it dives deep once again, and the curtains close.`
```

				goto end
			label seed
			`	Once you have the seed in your hands, the organizers tell you to go through the path on the left, and you do so.`
			`	The light you had seen turns out to be a bonfire, and you find a spot to sit on one of the logs close to it. You keep your eyes on the wooden stage in front of you, about a third of the size of human opera stages. Several lights shine onto it, and as you look up you can only barely see some spotlights high up in the trees. After the audience is all here, the curtains open to reveal many small figures made out of what looks like paper in a style that loosely resembles origami. They are supported either by thin wooden poles from below, or strings from above. There are figures close to where the curtains were, and others further into the stage, the various layers granting depth to the scenery. After a brief moment, the show begins.`

> 			`	The central figure is a young Saryd boy, who is galloping in a dense forest, as the sound effects of thunder grow increasingly frequent, echoing around you as if in a real thunderstorm. A white piece of paper, shaped like a lightning bolt, comes down swiftly, striking one of the trees in the boy's way, with flame-shaped folds raining down from the top of the stage and sticking to the tree as it falls, blocking the boy's path, so he turns to run elsewhere. Another lightning bolt, and another tree falls. After a few more repetitions, the boy is too tired to run anymore, hurt by some of the flames and now limping through the forest.`

Reviewer 3: The paragraph is much clearer now, but I still have some suggestions.
```suggestion
			`	The central figure is a young Saryd boy, who is galloping in a dense forest, as the sound effects of thunder grow increasingly frequent, echoing around you as if in a real thunderstorm. A white piece of paper, shaped like a lightning bolt, comes down swiftly and strikes one of the trees in the boy's way. Flame-shaped folds rain down from the top of the stage and stick to the tree as it falls, blocking the boy's path. He turns to run elsewhere. Another lightning bolt, and another tree falls. After a few more repetitions, the boy is too tired to run anymore, hurt by some of the flames and now limping through the forest.`
```
